### PR TITLE
Colour related links

### PR DESCRIPTION
**Pull request by madwort**
When a node is highlighted, colour all links connected to it red.
Should make it easier to see the connections when the graph is dense.

### CODE REVIEW

File: rhizome.js [hunk 1]
```diff
@@ -252,15 +267,16 @@ function highlightNode(target_id) {
     }
     idNodeOpacity(d.source.id);
     idNodeOpacity(d.target.id);
+    idLinkColor(d.source.id+"-"+d.target.id);
   });
 
   second_degree_nodes.forEach(function(n){
-    console.log('node',n);
```
Comment (madwort) [inline rhizome.js:258]: remove debug info

File: rhizome.js [hunk 2]
```diff
@@ -252,15 +267,16 @@ function highlightNode(target_id) {
...
     links.filter(function(d) {
       return ((d.source.id == n) || (d.target.id == n));
     }).forEach(function(d){
       idNodeOpacity(d.source.id);
       idNodeOpacity(d.target.id);
+      idLinkColor(d.source.id+"-"+d.target.id);
```
Comment (madwort) [inline rhizome.js:279]: second-order links could be a different colour...?

File: rhizome.js [hunk 3]
```diff
@@ -67,6 +67,10 @@ var zoom =
     g.attr("transform", "translate(" + d3.event.transform.x + "," +
           d3.event.transform.y + ")scale(" + d3.event.transform.k + ")");
   });
+
+var link_colour_default = '#999';
```
Comment (madwort) [inline rhizome.js:71]: these should probably be `linkColourDefault` instead of `link_colour_default` but it's so hideous...

**Comment by madwort** [comment]
rhizome-red-links

File: rhizome.js (hunk 2 above)
Comment (coreymwamba) [reply rhizome.js:279]: Yes!

File: rhizome.js (hunk 3 above)
Comment (coreymwamba) [reply rhizome.js:71]: JavaScript.

**Pull request merged**